### ISSUE TITLE
.glb vertex paint data not displaying correctly

### ISSUE THREAD

**Issue by Mangonels** [godotengine/godot#81821]
### Godot version

v4.1.1.stable.official [bd6af8e0e]

### System information

Godot v4.1.1.stable - Windows 10.0.19045 - Vulkan (Compatibility) - NVIDIA GeForce RTX 3050 Laptop GPU (NVIDIA; 31.0.15.2802) - 12th Gen Intel(R) Core(TM) i7-12700H (20 Threads)

### Issue description

.glb model with vertex paint data placed in 3D scene with forward+ shows stretched out coloring.
A default Surface0 material is generated with Vertex Color Use As Albedo true.

The blender export config only includes Vertex Data and Normals. Y up, -Z forward.

Importing a .obj file with the same data and dragging it to a 3D scene displays vertex colors correctly (an override material is to be added manually, and use vertex color as albedo toggled manually, in .obj's case)

![image](https://github.com/godotengine/godot/assets/24245302/d8be30a6-8474-492f-bbd5-32f5ece8c7e7)
forward+ 3D scene .obj + .glb side by side respectively using forward+ renderer and no lighting or ambience.

**Note:** The original issue was encountered in compatibility mode rendering, but later also reproduced in forward+ just to make sure it wasn't render mode specific. The sample project is forward+.

### Steps to reproduce

1. Generate a .glb file with a 3D model using various combined colors through vertex paint (a terrain mesh with mid-high vertex subdivision count is recommended). I did mine with Blender.

2. Import the .glb file into godot, and place it in a new 3D scene.

3. Notice a material is added to it automatically using vertex paint as albedo, but the coloring is stretched vs how it would be seen in blender, or as an .obj export.

### Minimal reproduction project

[Testing forward+.zip](https://github.com/godotengine/godot/files/12643433/Testing.forward%2B.zip)

**Comment by Calinou** [comment]
Can you reproduce this with **Generate LODs** disabled in the scene import settings on the .glb file?

**Comment by Mangonels** [comment]
The issue is still reproducible with Generate LOD's disabled and then reimporting, asuming this one:
![image](https://github.com/godotengine/godot/assets/24245302/7c83da38-d054-4454-9928-ebf9a9711b04)

As for vertex compression (asuming blender export one for .glb), it was already disabled for this example. I only export Normals and Vertex Color.

**Comment by Mangonels** [comment]
Blender 3.4 (the blender version I used) had a bug exporting the vertex paint data in .glb which seems fixed in Blender 3.6.
Furthermore, exporting .glb you don't even need to add a material to the mesh in Godot. Surface0 shows a preexisting material with use vertex paing as albedo setting already toggled true. This isn't the case with .obj, where you must add the material as an override manually (I believe this is already reported as a bug)

In essence, it seems this wasn't a Godot bug, but a Blender bug, which has already been fixed. Feel free to close this issue.
